fitlins/interfaces/bids.py

```diff
@@ -203,8 +203,20 @@ def _load_level1(self, runtime, analysis):
                 space = ents.pop('space')
             else:
                 # Picks first match
-                space = analysis.layout.get_spaces(suffix='bold',
-                                                   extensions=['.nii', '.nii.gz'])[0]
+                spaces = set(
+                    analysis.layout.get_spaces(suffix='bold',
+                                               extensions=['.nii', '.nii.gz'])
+                )
```

Comment:
Unclear what this being a `set` instead of a `list` gets us. Maybe just drop the `set()` wrapper?